mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/CassandraMailboxManagerTest.java

```diff
@@ -118,21 +118,21 @@ void setUp() throws Exception {
 
         @Test
         void deleteMailboxShouldUnreferenceMessageMetadata(CassandraCluster cassandraCluster) throws Exception {
-            ComposedMessageId composedMessageId = inboxManager.appendMessage(MessageManager.AppendCommand.builder()
+            AppendResult appendResult = inboxManager.appendMessage(MessageManager.AppendCommand.builder()
```

Comment:
I don't get when it did break

Reply:
(upon rebase)

As I chose to add test case one by one, then rebasing this change to previous commits is really long (error prone).